lib/execution_engine2/db/models/models.py

```diff
@@ -339,9 +339,13 @@ class Job(Document):
     meta = {"collection": "ee2_jobs"}
 
     def save(self, *args, **kwargs):
-        self.updated = time.time()
+        self.updated = datetime.utcnow().timestamp()
         return super(Job, self).save(*args, **kwargs)
 
+    def modify(self, query=None, **update):
+        update["updated"] = datetime.utcnow().timestamp()
+        return super(Job, self).modify(query=None, **update)
```

Comment:
Shouldn't this just pass the query as is, not pass `None`?

Reply:
Assuming I understand this I guess no code uses the query field otherwise tests should've broken

Reply:
Yea it should pass the query good call

Reply:
how did the tests pass with this bug?